libs/beacon/src/beacon_service.cpp

```diff
@@ -230,15 +230,15 @@ void BeaconService::StartNewCabinet(CabinetMemberList members, uint32_t threshol
   SharedAeonExecutionUnit beacon = std::make_shared<AeonExecutionUnit>();
 
   // Determines if we are observing or actively participating
-  if (members.find(identity_) == members.end())
+  if (members.find(identity_.identifier()) == members.end())
   {
     beacon->observe_only = true;
     FETCH_LOG_INFO(LOGGING_NAME, "Beacon in observe only mode. Members: ", members.size());
   }
   else
   {
     beacon->manager.SetCertificate(certificate_);
-    beacon->manager.Reset(members, threshold);
+    beacon->manager.NewCabinet(members, threshold);
```

Comment:
For consistent naming here I assume

Reply:
Not for consistent naming. It was because I introduced a new function Reset() for each time the DKG is re-run with the same cabinet. I changed the existing Reset to NewCabinet to clarify the different between resetting the beacon and instating a new cabinet.